packages/icons/src/icon/Icon.css

```diff
@@ -1,6 +1,6 @@
 /* Style applied to the root element */
 .saltIcon {
-  --icon-color: var(--saltIcon-color, var(--salt-text-secondary-foreground));
+  --icon-color: currentColor;
```

Comment:
--saltIcon-color is used by a lot of components e.g. Button, we can't just remove it here

Reply:
It's not removed

Reply:
We would also see things showing up in the visual diff if it was removed

Reply:
gotcha I missed that, didnt check chromatic :D